test/integration/test_integration.py

```diff
@@ -20,9 +20,9 @@
     DynamicallyPerAxisQuantizedLinear,
 )
 from torchao.quantization.quant_api import (
-    int4wo,
-    int8wo,
-    int8da_int8w,
+    int4_weight_only,
+    int8_weight_only,
+    int8_dynamic_act_int8_weight,
```

Comment:
can we expand out act into activation as well? Might as well since we hardly have any abbreviations left